src/vcbuild/dmd.vcxproj

```diff
@@ -216,6 +216,15 @@
   <ItemGroup>
     <ClInclude Include="../dmd/**/*.h" />
   </ItemGroup>
+  <ItemGroup>
+    <CustomBuild Include="..\..\VERSION">
+      <Message>Copying VERSION</Message>
+      <Command>echo on
+if not exist $(OutDir)\dmd\generated md $(OutDir)\dmd\generated
+if errorlevel 0 copy %(Identity) $(OutDir)\dmd\generated</Command>

```

Comment:
Would it be possible to run the config binary here?
https://github.com/dlang/dmd/blob/master/config.d

Reply:
> Would it be possible to run the config script here?

Yes, but it will not automatically update. Amended.